components/scroll-wrapper/README.md

```diff
@@ -0,0 +1,22 @@
+# Scroll Wrapper
+
+The `d2l-scroll-wrapper` element can be used to wrap content which may overflow its horizontal boundaries, optionally providing left/right scroll buttons.
+
+![scroll wrapper](./screenshots/scroll-wrapper.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/scroll-wrapper/scroll-wrapper.js';
+</script>
+<d2l-scroll-wrapper show-actions>
+	<!-- content which may overflow horizontall -->
+</d2l-scroll-wrapper>
+```
+
+**Properties:**
+
+| Property | Type | Description |
+|--|--|--|
+| `show-actions` | Boolean, default: `false` | Whether to show left/right scroll buttons |
```

Comment:
It's probably not worth updating everywhere using table to switch it, but it's always felt weird to me that this functionality defaults to `false`, instead of using a `hide-actions` property.

Reply:
I agree, and I've toiled with flipping this around as well. It wouldn't affect places using `d2l-table`, just those using `d2l-scroll-wrapper` directly... which according to code search is:
- 5 places that are setting `show-actions`
- 3 places that are not (2 of which are in the monolith and have been broken since last summer)

I think I will do this... it can be pretty safe as I can go into those 3 places right now and add `hide-actions` and then things will "just work" when the switchover happens.

Reply:
Alright, I've flipped this around to `hide-actions` and opened a [PR into the LMS](https://github.com/Brightspace/lms/pull/9856) and [manager view](https://github.com/Brightspace/manager-view-fra/pull/2123) to add that attribute into the 2 places that wanted the actions hidden.